docs/en/user_guides/useful_tools.md

```diff
@@ -2,84 +2,107 @@
 
 Apart from training/testing scripts, We provide lots of useful tools under the `tools/` directory.
 
-## Analysis Tools
-
 <!-- TOC -->
 
-- [Log Analysis](#log-analysis)
-- [Model Complexity (experimental)](#model-complexity-experimental)
-- [Benchmarks (experimental)](#benchmarks)
-- [Print the entire config](#print-the-entire-config)
+- [Analysis Tools](#analysis-tools)
+  - [Log Analysis](#log-analysis)
+  - [Model Complexity (Experimental)](#model-complexity-experimental)
+  - [Print the entire config](#print-the-entire-config)
+- [Dataset Tools](#dataset-tools)
+  - [Animal Pose](#animal-pose)
+  - [COFW](#cofw)
+  - [DeepposeKit](#deepposekit)
+  - [Macaque](#macaque)
+  - [Human36M](#human36m)
+  - [MPII](#mpii)
```

Comment:
How about removing the catalog? Since there is 'Shortcuts' on readthedocs pages
![image](https://user-images.githubusercontent.com/26127467/188544708-33aee011-4ef2-44df-97e6-c7252c8b6f7f.png)

Reply:
I think the catalog is useful when users read the original markdown doc in Github directly.